pkg/metrics/otel.go

```diff
@@ -184,51 +184,51 @@ processors:
           - action: aggregate_labels
             # label_set is the allowlist of labels to keep after aggregation
             label_set: [status, operation]
-            aggregation_type: sum
+            aggregation_type: max
       - include: declared_resources
         action: update
         new_name: current_declared_resources
         operations:
           - action: aggregate_labels
             label_set: []
-            aggregation_type: sum
+            aggregation_type: max
       - include: kcc_resource_count
         action: update
         operations:
           - action: aggregate_labels
             label_set: [resourcegroup]
-            aggregation_type: sum
+            aggregation_type: max
       - include: reconciler_errors
         action: update
         new_name: last_reconciler_errors
         operations:
           - action: aggregate_labels
             label_set: [component, errorclass]
-            aggregation_type: sum
+            aggregation_type: max
       - include: reconcile_duration_seconds
         action: update
         operations:
           - action: aggregate_labels
             label_set: [status]
-            aggregation_type: sum
+            aggregation_type: max
       - include: rg_reconcile_duration_seconds
         action: update
         operations:
           - action: aggregate_labels
             label_set: [stallreason]
-            aggregation_type: sum
+            aggregation_type: max
       - include: last_sync_timestamp
         action: update
         operations:
           - action: aggregate_labels
             label_set: [status]
-            aggregation_type: sum
+            aggregation_type: max
       - include: parser_duration_seconds
         action: update
         operations:
           - action: aggregate_labels
             label_set: [status, source, trigger]
-            aggregation_type: sum
+            aggregation_type: max
       - include: pipeline_error_observed
         action: update
         new_name: last_pipeline_error_observed
```

Comment:
this is a timestamp, right? should it be max?

Reply:
Not a time stamp, the value is a boolean, records 1 if errlen > 0 https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/metrics/record.go#L73-L74

Reply:
ah. then yeah, we definitely want max, not sum.

Reply:
max it is.